packages/graphql/src/mutations/_txHelper.js

```diff
@@ -21,6 +24,12 @@ export async function checkMetaMask(from) {
 // to hang
 const isServer = typeof window === 'undefined'
 
+// function useRelayer({ mutation }) {
```

Comment:
Is the plan to gate this with an env var as well ? like ENABLE_RELAYER ?

Reply:
Yep